assets/js/advisorForm.js

```diff
@@ -45,6 +47,11 @@ for (let i = 0; i < buttonNext.length; i += 1) {
                 }, 200);
             }
         } else {
+            if (questions[i] === questions[8]) {
```

Comment:
Don't understand why you have to make exception for number 8 and 15 (I know we saw that this afternoon but don't understand the logic), can we check it tomorrow morning